.travis.yml

```diff
@@ -2,12 +2,12 @@ language: node_js
 node_js:
   - "stable"
 addons:
-  firefox: "47.0.1"
+  firefox: "50.0.2"

```

Comment:
I suspect this change is causing the travis-ci build failure. Why did you update to the latest Firefox? I was stuck on 47x because the later versions didn't support loading unsigned add-ons. Has that been fixed?